interfaces/Portalicious/src/app/pages/projects-overview/project-summary-card/project-summary-card.component.html

```diff
@@ -0,0 +1,110 @@
+<ng-template
+  #metric
+  let-value="value"
+  let-label="label"
+>
+  <dd class="px-2 py-1 font-bold">
+    @switch (metrics.status()) {
+      @case ('pending') {
+        <p-skeleton
+          class="my-1 cursor-progress text-lg"
+          height="1.125rem"
+          [width]="randomWidth"
+        />
+      }
+      @case ('success') {
+        {{ value }}
+      }
+    }
+  </dd>
+  <dt class="px-2 py-1 text-sm">
+    {{ label }}
+  </dt>
+</ng-template>
+
+<p-card
+  styleClass="group relative h-full border border-transparent *:h-full focus-within:border-primary hover:border-primary [&_.p-card-title]:flex-grow [&_.p-card-content]:flex-end"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]="['/', AppRoutes.project, id()]"
+      class="after:absolute after:inset-0 after:block group-focus-within:text-primary group-hover:text-primary"
```

Comment:
The classes "`after:absolute after:inset-0 after:block`" are the trick that make the whole Card clickable, without any javascript-magic.

Reply:
But I don't have the Prettier/Tailwind issue locally about the order of the class-names.. :/ ?

Reply:
That's odd....